packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PickerItemsDefault.scss

```diff
@@ -2,23 +2,33 @@
 
 .personaContainer {
   @include focus-border();
+  border-radius: 15px;
   display: inline-flex;
   align-items: center;
   background: $ms-color-neutralLighter;
   margin: 1px;
   cursor: default;
   user-select: none;
-  max-width: 100%;
+  max-width: 180px;
```

Comment:
Yeah, I think setting an exact width (as Kris has done) is our best option here.